packages/@okta/vuepress-site/docs/reference/api/apps/index.md

```diff
@@ -9,7 +9,7 @@ The Okta Application API provides operations to manage applications and/or assig
 
 ## Get started
 
-Explore the Apps API: [![Run in Postman](https://run.pstmn.io/button.svg)](https://app.getpostman.com/run-collection/be7367401f4926a945f8)
+Explore the Apps API: [![Run in Postman](https://run.pstmn.io/button.svg)](https://app.getpostman.com/run-collection/a1baf2ec75f797e6af36)
```

Comment:
![image](https://user-images.githubusercontent.com/9053598/137544588-dea4c688-0c55-46bb-b649-6b4bb5e1aac7.png)
The header should be `SSWS {{apikey}}` to keep it consistent with the rest of the collection.

Reply:
Is this particular to updating the client? Can you not add the profile attributes during creation? If you can during creation, wouldn't it make sense to re-use the existing request under `Add Application` -> `Add OAuth2 Client App`?

Reply:
1. good catch, modified header value to SSWS {{apikey}}
2. As discussed, one cannot add it through UI. After discussing in [jira](https://oktainc.atlassian.net/browse/OKTA-426644?focusedCommentId=2191540) It looks like this a popular request from customers, so I modified Add App OIDC Api to have profile attributes, also included Assign Profile Attributes API (Update Apps API) to let customers have the option to include profile attributes after app creation.